### ISSUE TITLE
How to represent that Kubernetes resources are not accessible?

### ISSUE THREAD

**Issue by feloy** [containers/podman-desktop#10385]
### UX Description

With https://github.com/podman-desktop/podman-desktop/issues/7629, the frontend now knows which Kubernetes resources (pods, deployments, nodes, etc) the user can access.

How do we want to represent that some resources are not accessible, in these different pages?

- In this page, Pods and/or Deployments may not be accessible:

![Image](https://github.com/user-attachments/assets/84a35ba4-a6c7-46dc-bcee-f65283e55cd5)

- In the Kubernetes menu, some Kubernetes resources may not be accessible:
- In the Kubernetes Dashboard, some Kubernetes resources may not be accessible:

![Image](https://github.com/user-attachments/assets/db8c1a93-9dd5-42f1-8510-2d0d38d3205d)

The backend sends two information for each Kubernetes resource:
- is the resource accessible (more exactly, does the user have the permission to watch the resource, used to get the list of objects)?
- optionally, a reason why it is accessible / not accessible

For now, the permissions to create/delete/... (all other operations except watch/list) are not known.

### Request type

UX analysis/suggestions for improvement

### Primary Contact

@feloy 

### Deadline for request

as part of the current sprint/next sprint

**Comment by MariaLeonova** [comment]
@feloy there are several ways to mark something inaccessible:
 - grey out a particular element
 - add an icon / badge; potentially accompanied by a small label text (`not accessible`)
 - display an error message next to it

There also needs to be a way to provide more info for the user, in form of a tooltip, for example on what the status is and why. 

@ekidneyrh the dashboard is your design, WDYT? Let's discuss!

**Comment by feloy** [comment]
> [@feloy](https://github.com/feloy) there are several ways to mark something inaccessible:
> 
> * grey out a particular element
> * add an icon / badge; potentially accompanied by a small label text (`not accessible`)
> * display an error message next to it
> 
> There also needs to be a way to provide more info for the user, in form of a tooltip, for example on what the status is and why.
> 

For the first page (the list of Kubernetes contexts), I was thinking about greying the `Pods` and/or `Deployments` text, and displaying a lock (🔒) instead of the number of pods/resources, with a tooltip explaining that the user doesn't have the authorization. That seems aligned with your proposal.


> [@ekidneyrh](https://github.com/ekidneyrh) the dashboard is your design, WDYT? Let's discuss!

**Comment by ekidneyrh** [comment]
I agree that having the #pods’ and ‘deployments’ grayed out would work on the list of kubernetes contexts. Instead of a lock though I would suggest ‘ - ‘ in replacement for the numbers. And then include an icon at the bottom with a toolip like Masha suggested. 

Here’s how that would look:

![Image](https://github.com/user-attachments/assets/382128a8-fd72-43ee-be4d-b400cb343c52)

For the Kubernetes dashboard we could have the same and also maybe grey out the icon too?

![Image](https://github.com/user-attachments/assets/3eba03c4-2908-4e4e-8b51-b4efbc3912cf)

For the dashboard would the user not be able to click into the deployments if it's inaccessible? If they are still clickable we can have a message appear on the pod or deployment page and the 'Apply YAML' button grayed out 

![Image](https://github.com/user-attachments/assets/28154725-367a-4a63-a8a2-b9cf666c26fc)

**Comment by feloy** [comment]
Great, thanks @ekidneyrh for these suggestions. I'll work on applying them
